numpy/ma/extras.py

```diff
@@ -1532,7 +1532,7 @@ def flatnotmasked_edges(a):
 
     See Also
     --------
-    flatnotmasked_contiguous, notmasked_contiguous, notmasked_edges,
+    flatnotmasked_contiguous, notmasked_contiguous, notmasked_edges
```

Comment:
The [rendered document](https://6741-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.ma.flatnotmasked_edges.html?highlight=notmasked_edges#numpy.ma.flatnotmasked_edges) adds the comma back? Nice.